flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
That's not needed since Electron is available for all arches supported by flathub.
```suggestion
```

Reply:
I'm aware of it, but the application's author only releases (and maybe supports) `x86_64` builds, so I'm not sure if we should go all-arches anyway.

Reply:
The app itself doesn't contain native code, so it shouldn't care about architecture, I guess.

Reply:
Yeah, it should be fine.

Reply:
Is it better to have a `flathub.json` with an empty object, or remove it altogether?